### ISSUE TITLE
the original function filterblq argument was lost when we refactored the code

### ISSUE THREAD

**Issue by smouksassi** [certara/tidyvpc#22]
https://github.com/olivierbarriere/vpcstats/commit/c691c3139691c23b492fcf3267b508e5a6fef67e#diff-1384b5ce45adcab039332f3a6cb6d613e1d6c01423d176ec92fe6f38eec9e9c2R43

this option can be useful when we want to equally filter out blq on obs and sim data before applying pred correction or further manipulation

opening this issue for sake of discussion on the best way forward

**Comment by PhilP86** [comment]
Indeed, this would be useful to include it as an argument in the censoring() function so that it lets the user decide whether observed bql should be set to -Inf or whether all bql (obs and sim) should be 'filtered out' by setting them to NA (setting them to NA instead of discarding them permitting to ensure dataframes for obs and all simulated replicates have the same dimension).

**Comment by billdenney** [comment]
With a quick thought on this (not looking through how things are handled in the code), here is what I would have expected using the functions provided:

I would expect the `censoring()` function to handle all BLQ (or ALQ) values and make them work correctly in any VPC that comes out of `vpcstats()`.  In other words, I would think that `filterblq` would be implicit in using the `censoring()` function.

Is there a time when `censoring()` would be used that `filterblq` would be desired to be set to `FALSE`?  (I can't think of one, but I've only thought about it for a few minutes.)

Generally, I think that if I give `censoring()` the desired output would have both the VPC for the continuous variable and the VPC for the fraction BLQ/ALQ by time would be provided.
